SPECS/git/git.spec

```diff
@@ -1,7 +1,13 @@
+
+%bcond daemon 1
+%bcond subtree 1
+%bcond svn 0
```

Comment:
Why are we dropping the `svn` subpackage in this PR? I doubt anyone's using it, but seems unrelated to updating `systemd`.

Reply:
"circular dependencies" problems.